src/Mono.Android/Mono.Android.targets

```diff
@@ -12,10 +12,9 @@
       Outputs="$(OutputPath)\..\v1.0\Java.Interop.dll">
     <MSBuild
```

Comment:
Ideally, we shouldn't need this `<MSBuild/>` invocation anymore, nor the `_BuildJavaInterop` target, nor the `@(Assembly)` item group (below).

Instead -- ideally -- we'd have a `@(ProjectReference)` within `Mono.Android.csproj` referring to `Java.Interop.csproj`, and MSBuild would handle it "normally".

Reply:
OK, I will try it in the next PR.